Action.c

```diff
@@ -410,6 +410,25 @@ static Htop_Reaction actionStrace(State* st) {
    return HTOP_REFRESH | HTOP_REDRAW_BAR;
 }
 
+static Htop_Reaction actionDebugger(State* st) {
+   Settings* settings = st->settings;
+   Process* p = (Process*) Panel_getSelected(st->panel);
+   if (!p) return HTOP_OK;
+
+   char buf[32];
+   xSnprintf(buf, sizeof(buf), "%s -p %d", settings->debuggerTool, (int) p->pid);
+
+   endwin();
+   int result = system(buf);
```

Comment:
This will misbehave on the off chance there are special characters in `settings->debuggerTool` -- `execvp()` is probably a more appropriate mechanism here I'd think.

Reply:
You're right; my `system()`-based solution was good back when I had `gdb` hard-coded, but now something better is needed.

Reply:
Done